### DISCUSSION TITLE
When specifying the font, how can I specify what weight to use? (level of boldness)

### DISCUSSION THREAD

**Discussion by avisra** [SixLabors/Fonts#402] [Q&A]
I see options for Bold font style - but I don't see anything about specifying weights...

**Comment by JimBobSquarePants** [comment]
Individual fonts don’t actually contain multiple weights. Here’s a great discussion on the matter.

https://stackoverflow.com/questions/39356906/do-font-files-e-g-ttc-ttf-otf-contain-different-weights

**Comment by avisra** [comment]
This was very helpful, thank you! So if I'm using one of the variable fonts from Google Fonts, are those not supported right now or I just need to split to separate ttf?

**Reply by JimBobSquarePants** [reply]
Not supported yet I'm afraid. Some of the work to enable support has started in https://github.com/SixLabors/Fonts/pull/342 as CFF and TTF share tables.
